doc/user/isisd.rst

```diff
@@ -594,6 +594,40 @@ The following command show Flex-Algo information:
 includes an 'algorithm (128-255)' optional argument. See
 :ref:`showing-isis-information` and :ref:`isis-segment-routing`.
 
+.. _isis-srv6:
+
+Segment Routing over IPv6 (SRv6)
+================================
+
+This feature enables extensions in IS-IS to support Segment Routing over IPv6
+data plane (SRv6) as per RFC 9352.
+
+.. clicmd:: segment-routing srv6
+
+   Enable Segment Routing over IPv6 data plane (SRv6).
+
+.. clicmd:: locator NAME
+
+   Specify the SRv6 locator to use for SRv6. The locator must be configured in
+   Zebra. Once the locator is configured, IS-IS automatically allocates prefix
+   SID and adjacency SIDs, creates local SID entries in the data plane, and
+   advertises them in the IGP domain.
+
+.. clicmd:: interface NAME
```

Comment:
I would expect `interface Name` under zebra to make this interface available not only for ISIS, but for other protocols?